com.unity.render-pipelines.core/Runtime/Textures/TextureXR.cs

```diff
@@ -38,6 +38,9 @@ public static bool useTexArray
                 {
                     case GraphicsDeviceType.Direct3D11:
                     case GraphicsDeviceType.Direct3D12:
+                    case GraphicsDeviceType.GameCoreScarlett:
+                    case GraphicsDeviceType.GameCoreXboxOne:
+                        return true;
```

Comment:
The return true shouldn't be in the middle here as that is what the whole list here is for.
So now it says
case MSPLATFORMS
    return true;
case OTHERVRPLATFORMS
    return true;
default:
     return false